Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -272,7 +273,28 @@ private void AnalyzeFile(string filePath)
             IEnumerable<Ast> funcDefAsts;
 
             // Use a List of KVP rather than dictionary, since for a script containing inline functions with same signature, keys clash
-            List<KeyValuePair<CommandInfo, IScriptExtent>> cmdInfoTable = new List<KeyValuePair<CommandInfo, IScriptExtent>>();            
+            List<KeyValuePair<CommandInfo, IScriptExtent>> cmdInfoTable = new List<KeyValuePair<CommandInfo, IScriptExtent>>();
+
+            //Check wild card input for the Include/ExcludeRules and create regex match patterns
+            List<Regex> includeRegexList = new List<Regex>();
+            List<Regex> excludeRegexList = new List<Regex>();
+            if (includeRule != null)
+            {
+                foreach (string rule in includeRule)
+                {
+                    Regex includeRegex = new Regex(rule.Replace("*", ".*?"), RegexOptions.IgnoreCase);
```

Comment:
I would suggest using this:
Regex reg = new Regex(String.Format("^{0}$", Regex.Escape(rule).Replace(@"\*", ".*")))